apps/dcos_l4lb/test/dcos_l4lb_lashup_vip_listener_SUITE.erl

```diff
@@ -70,6 +70,7 @@ lookup_failure3(Config) ->
 
 lookup_vip(_Config) ->
   {ok, _} = update_de8b9dc86_marathon(),
+  timer:sleep(100),
```

Comment:
does it that long?

Reply:
Well, it's hard to say. We don't have metric for this. It's a flaky test. What value do you think we should use here?

Reply:
Sorry, I was under impression that the input to `sleep` is in seconds. Looks like it's in milliseconds. We should be fine then